packages/teleport-component-generator-html/src/plain-html-mapping.ts

```diff
@@ -23,6 +23,19 @@ export const PlainHTMLMapping: Mapping = {
         },
       },
     },
+    'date-time-node': {
+      elementType: 'date-time-primitive',
+      dependency: {
+        type: 'package',
+        version: '1.0.0',
+        path: 'date-time-primitive',
```

Comment:
Just dropping by, when we publish the package to `npm` this should be changed to
`@teleporthq/date-time-primitive`